weaviate/collection/classes/data.py

```diff
@@ -17,15 +17,15 @@ def to_include(self) -> str:
         return ",".join(include)
 
 
-class GetObjectByIdMetadata(IncludesModel):
-    classification: bool = False
-    vector: bool = False
+class _GetObjectByIdMetadata(_IncludesModel):
+    classification: bool = Field(default=False)
+    vector: bool = Field(default=False)
 
 
-class GetObjectsMetadata(IncludesModel):
-    classification: bool = False
-    featureProjection: bool = Field(False, alias="feature_projection")
-    vector: bool = False
+class _GetObjectsMetadata(_IncludesModel):
```

Comment:
should we completly remove them?

Reply:
Yeh, did this as a first run to see feedback

Will `classification` or `featureProjection` ever be supported? If not, I'll get rid!

Reply:
I don't know - I have no idea if anybody is acutally using this. So I'd say let's remove it for now and see if anyone complains. But I don't feel strongly about this